webapp/platform/types/src/client4.ts

```diff
@@ -37,3 +37,23 @@ export type FetchPaginatedThreadOptions = {
     fromCreateAt?: number;
     fromPost?: string;
 }
+
+export enum ReportDuration {
+    Last30Days = 'last_30_days',
+    PreviousMonth = 'previous_month',
+    Last6Months = 'last_6_months',
+}
+
+export type UserReportOptions = {
+    sort_column: 'CreateAt' | 'Username' | 'FirstName' | 'LastName' | 'Nickname' | 'Email',
+    page_size: number,
+    sort_direction?: 'asc' | 'desc',
+    date_range?: ReportDuration,
+    last_column_value?: string,
+    last_id?: string,
+    role_filter?: string,
```

Comment:
what are the values expected from the client to be passed here, i could find
`system_admin` & `system_guest` but not member

Reply:
may be we can define these here?

Reply:
Oh i see, but is it there for this version?